environment-model/environment-model_shacl.ttl

```diff
@@ -3,134 +3,286 @@
-            sh:description "number of tiles as row"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of tile_count_x failed!"@en ;
-            sh:minCount "0"^^xsd:unsignedInt ;
-            sh:name "tile_count_x"@en ;
-            sh:path environment-model:environment-model_quantity_tile_count_x ],
-        [ skos:example "Trian3DBuilder, version 7.7" ;
-            sh:datatype xsd:string ;
-            sh:description "tool for the creation of the data"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of creation_source failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "creation_source"@en ;
-            sh:path environment-model:environment-model_project_creation_source ],
-        [ skos:example "SDK 4.25" ;
+            sh:description "Ellipsodial height or orthometric height"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of heightSystem failed!"@en ;
+            sh:name "heightSystem"@en ;
+            sh:order 1 ;
+            sh:path environment-model:heightSystem ] ;
+    sh:targetClass environment-model:GeodeticReferenceSystem .
+
+environment-model:BoundingBoxShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:float ;
+            sh:message "Validation of yMin failed!"@en ;
+            sh:name "yMin"@en ;
+            sh:maxCount 1 ;
+            sh:order 1 ;
+            sh:path environment-model:yMin ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of yMax failed!"@en ;
+            sh:name "yMax"@en ;
+            sh:maxCount 1 ;
+            sh:order 3 ;
+            sh:path environment-model:yMax ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of xMin failed!"@en ;
+            sh:name "xMin"@en ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path environment-model:xMin ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of xMax failed!"@en ;
+            sh:name "xMax"@en ;
+            sh:maxCount 1 ;
+            sh:order 2 ;
+            sh:path environment-model:xMax ] ;
+    sh:targetClass environment-model:BoundingBox .
+
+environment-model:Coordinate2DShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:float ;
+            sh:message "Validation of x failed!"@en ;
+            sh:name "x"@en ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path environment-model:x ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of y failed!"@en ;
+            sh:name "y"@en ;
+            sh:maxCount 1 ;
+            sh:order 1 ;
+            sh:path environment-model:y ] ;
+    sh:targetClass environment-model:Coordinate2D .
+    
+environment-model:ProjectLocationShape a sh:NodeShape ;
+    sh:property [ skos:example "nan" ;
+            sh:datatype xsd:string ;
+            sh:description "State of project area"@en ;
```

Comment:
@jtdemer we have ISO standards for this!

https://en.wikipedia.org/wiki/ISO_3166-2


Please rework all values according to proper ISO definitions if avaliable

Reply:
outdated. 
moved to Georeference shacl . see georeference:country